tests/test_clientsdaily.py

```diff
@@ -22,27 +25,29 @@ def make_frame(spark):
 
 
 @pytest.fixture
-def make_frame_with_extracts(spark):
-    from mozetl.clientsdaily import rollup
-    frame = make_frame(spark)
-    return rollup.extract_search_counts(frame)
+def main_summary_with_search(main_summary):
+    return cd.extract_search_counts(main_summary)
 
 
-def test_extract_search_counts(spark):
-    from mozetl.clientsdaily import rollup
+@pytest.fixture
+def clients_daily(main_summary_with_search):
+    return cd.to_profile_day_aggregates(main_summary_with_search)
+
 
-    frame = make_frame(spark)
-    extracted = rollup.extract_search_counts(frame)
-    row = extracted.agg({'search_count_all': 'sum'}).collect()[0]
+def test_extract_search_counts(main_summary_with_search):
+    row = main_summary_with_search.agg({'search_count_all': 'sum'}).collect()[0]
     total = row.asDict().values()[0]
```

Comment:
This could also be done with `df.first()[0]`, where `.first()` returns a `Row`, and `Row[0]` returns the first item.